messages/en.json

```
@@ -294,16 +294,99 @@
+  "credits-pricing-storage-tier-next-38gib-price": "$0.05 per GiB",
+  "credits-pricing-storage-tier-next-187gib": "Next 187 GiB",
+  "credits-pricing-storage-tier-next-187gib-price": "$0.04 per GiB",
+  "credits-pricing-storage-tier-next-390gib": "Next 390 GiB",
+  "credits-pricing-storage-tier-next-390gib-price": "$0.03 per GiB",
+  "credits-pricing-storage-tier-next-640gib": "Next 640 GiB",
+  "credits-pricing-storage-tier-next-640gib-price": "$0.025 per GiB",
+  "credits-pricing-storage-tier-over-1tb": "Over 1 TB",
+  "credits-pricing-storage-tier-over-1tb-price": "$0.021 per GiB",
+  "credits-pricing-footnote": "* Storage is calculated per GiB per hour.",
+  "credits-pricing-disclaimer": "Credits cover usage beyond included plan limits. Credits are prepaid and remain valid for 12 months.",
+  "no-credits-available": "No credits yet. Purchase a pack to extend your plan usage.",
```

Biome (2.1.2)

[error] The key no-credits-available was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)